src/autolabel/labeler.py

```diff
@@ -127,7 +127,7 @@ def run(
         for current_index in track_with_stats(
             indices,
             postfix_dict,
-            total=len(inputs),
+            total=len(inputs) - current_index,
```

Comment:
should this be "total=len(inputs) - current_index - 1"?

Reply:
if current_index is the number of elements that have been completed so far?

Reply:
Current index is the latest example which has *not* been labeled. So, if the first five examples have been labeled, the index would be 5 (0 indexing)

Reply:
ah okay never mind then!